### PR TITLE
Allow non-default port database importing

### LINKED ISSUE #620: Postgresql port parameter is ignored.
## Affected Puppet, Ruby, OS and module versions/distributions

- Puppet: 5.5.10
- Ruby: 2.5.1p57
- Distribution: Ubuntu 18.04.2
- Module version: 6.7.0

## How to reproduce (e.g Puppet code you use)
database_port => '5444' 
## What are you seeing
Parameter is ignored as default port 5432 is hard coded:
`
manifests/database/postgresql.pp:    command => "echo ${database_host}:5432:${database_name}:${database_user}:${database_password} >> /root/.pgpass",
`

## What behaviour did you expect instead
Parameter "database_port => '5444' " to be used when building "/root/.pgpass" and to be used when executing "Exec[zabbix_server_create.sql]"
`
manifests/database/postgresql.pp:    command => "echo ${database_host}:${database_port}:${database_name}:${database_user}:${database_password} >> /root/.pgpass",
manifests/database/postgresql.pp:    unless  => "grep \"${database_host}:${database_port}:${database_name}:${database_user}:${database_password}\" /root/.pgpass",
`

## Output log
`
Error: 'cd /usr/share/doc/zabbix-*-pgsql && if [ -f create.sql.gz ]; then gunzip -f create.sql.gz ; fi && psql -h 'zabbix.db.domain.com' -U 'zabbix' -d 'zabbix' -f create.sql && touch /etc/zabbix/.schema.done' 
returned 2 instead of one of [0]
Error: /Stage[main]/Zabbix::Database::Postgresql/Exec[zabbix_server_create.sql]/returns: change from 'notrun' to ['0'] failed: 'cd /usr/share/doc/zabbix-*-pgsql && if [ -f create.sql.gz ]; then gunzip -f create.
sql.gz ; fi && psql -h 'zabbix.db.domain.com' -U 'zabbix' -d 'zabbix' -f create.sql && touch /etc/zabbix/.schema.done' returned 2 instead of one of [0] (corrective)
`

## Any additional information you'd like to impart
Command "Exec[zabbix_server_create.sql]" should be updated as well **-p "${database_port}"**:
`
cd /usr/share/doc/zabbix-*-pgsql && if [ -f create.sql.gz ]; then gunzip -f create.sql.gz ; fi && psql -h 'zabbix.db.domain.com' -U 'zabbix' -d 'zabbix' -p "${database_port}" -f create.sql && touch /etc/zabbix/.schema.done
`

### PR DESCRIPTION
**Pull request by root-expert**
#### Pull Request (PR) description
Importing database with non-default port (for both mysql and postgres) was failing.

#### This Pull Request (PR) fixes the following issues
Fixes #620

### CODE REVIEW

File: manifests/database/mysql.pp [hunk 1]
```diff
@@ -23,6 +23,7 @@
   $database_user        = '',
   $database_password    = '',
   $database_host        = '',
+  $database_port        = 3306,
```
Comment (bastelfreak) [inline manifests/database/mysql.pp:26]: please add a datatype here and some inline documentation (I know that the other params aren't document at the moment :()
I suggest [Stdlib::Port::Unprivileged](https://github.com/puppetlabs/puppetlabs-stdlib/blob/main/types/port/unprivileged.pp)
```suggestion
  Stdlib::Port::Unprivileged $database_port = 3306,
```

File: manifests/database/postgresql.pp [hunk 2]
```diff
@@ -23,6 +23,7 @@
   $database_user        = '',
   $database_password    = '',
   $database_host        = '',
+  $database_port        = 5432,
```
Comment (bastelfreak) [inline manifests/database/postgresql.pp:26]:
```suggestion
  Stdlib::Port::Unprivileged $database_port = 5432,
```

File: manifests/database/mysql.pp (hunk 1 above)
Comment (root-expert) [reply manifests/database/mysql.pp:26]: Thanks! What do you mean by "inline documentation"? Like next to the parameter?

Comment (bastelfreak) [reply manifests/database/mysql.pp:26]: in the header of the file, like in https://github.com/voxpupuli/puppet-zabbix/blob/master/manifests/template.pp :)

Comment (root-expert) [reply manifests/database/mysql.pp:26]: Do you want me to document them all as we are here?

**Comment by root-expert** [comment]
Rebased with master as well.

**Review by bastelfreak — APPROVED** [review]

**Review by bastelfreak — APPROVED** [review]

**Review by bastelfreak — APPROVED** [review]

File: manifests/database/mysql.pp (hunk 1 above)
Comment (bastelfreak) [reply manifests/database/mysql.pp]: if you want to do that, that would be awesome. In that caee please provide them as puppet-strings style

**Pull request merged**